etc/kayobe/stackhpc.yml

```diff
@@ -130,6 +130,23 @@ stackhpc_kolla_source_version: "stackhpc/{{ openstack_release }}"
 stackhpc_kolla_ansible_source_url: "https://github.com/stackhpc/kolla-ansible"
 stackhpc_kolla_ansible_source_version: "stackhpc/{{ openstack_release }}"
 
+###############################################################################
+# Overcloud host image source
```

Comment:
Wondering if these should be in a separate file. `stackhpc-overcloud-host-images.yml`?

Reply:
Done